### DISCUSSION TITLE
Trying to make a trivial localization fix

### DISCUSSION THREAD

**Discussion by amire80** [oppia/oppia-android#5256] [Q&A (Navigating the codebase)] [answered]
Hi!

I'm trying to make a trivial fix in the English strings file to prevent subtle code validation errors on translatewiki, where Oppia is localized. Here's the pull request: 

https://github.com/oppia/oppia-android/pull/5255

The actions seem to be stuck. Is there anything I can do to get this reviewed? It's supposed to be very simple.

Thanks!

**Comment by seanlip** [comment, marked as answer]
Hi, I think actions on this repo need to be approved for new contributors. I've kicked them off -- thanks for flagging! 

Closing this but please feel to reopen it if there are further blockages. Thanks for your PR!
